e2e/launcher/configuration.go

```diff
@@ -0,0 +1,52 @@
+package launcher
+
+import (
+	"encoding/json"
+	"os"
+	"path/filepath"
+)
+
+type NetworkConfiguration struct {
+	NetworkId          string               `json:"network_id"`
+	GenesisPath        string               `json:"genesis_path"`
+	NodeConfigurations []*NodeConfiguration `json:"node_configurations"`
+}
+
+type NodeConfiguration struct {
+	PrivateKey string `json:"private_key"`
+	ConfigPath string `json:"config_path"`
+}
+
+const networkConfigurationFileName = "network.json"
+
+func loadNetworkConfiguration(networkConfigDirectory string) (*NetworkConfiguration, error) {
```

Comment:
```suggestion
func loadNetworkConfiguration(networkConfigDirectory string) (networkConfiguration *NetworkConfiguration, err error) {
```

Given that you have to declare these individually, consider using named return values

Reply:
I see an 👀 , but no comments/resolution/change. What does this mean?